src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -146,6 +150,7 @@ public ModelAndView pushGradesToCanvas(@PathVariable Long sectionId, @ModelAttri
                 page.addObject("pushingSuccessful", true);
             }
             catch (AttendanceAssignmentException e){
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)